src/components/map/ha-map.ts

```diff
@@ -469,26 +477,30 @@ export class HaMap extends ReactiveElement {
       }
       #map.dark {
         background: #090909;
-        --map-filter: invert(0.9) hue-rotate(170deg) grayscale(0.7);
+        color: #ffffff;
+        --map-filter: invert(0.9) hue-rotate(170deg) brightness(1.5)
+          contrast(1.2) saturate(0.3);
+      }
+      #map.light {
+        background: #ffffff;
+        color: #000000;
+        --map-filter: invert(0);
```

Comment:
What does this do? `--map-filter: unset;`

Reply:
This forces the map to appear in light mode when HA is in dark mode.  With `--map-filter: unset;` the map stays in dark mode.